src/NetAppFiles/NetAppFiles/help/Update-AzNetAppFilesVolume.md

```diff
@@ -0,0 +1,276 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.NetAppFiles.dll-Help.xml
+Module Name: Az.NetAppFiles
+online version: https://docs.microsoft.com/en-us/powershell/module/az.netappfiles/update-aznetappfilesvolume
+schema: 2.0.0
+---
+
+# Update-AzNetAppFilesVolume
+
+## SYNOPSIS
+Updates an Azure NetApp Files (ANF) volume according to the optional modifiers provided.
+
+## SYNTAX
+
+### ByFieldsParameterSet (Default)
+```
+Update-AzNetAppFilesVolume -ResourceGroupName <String> -Location <String> -AccountName <String>
```

Comment:
Same comment about Location applies to all updates - you should need it to get the resource

Reply:
Same answer.